OSXvnc-server/main.c

```diff
@@ -1105,6 +1101,10 @@ BOOL runningLittleEndian ( void ) {
 
 int main(int argc, char *argv[]) {
     NSAutoreleasePool *tempPool = [[NSAutoreleasePool alloc] init];
+    static dispatch_once_t onceStreamingInit;
+    dispatch_once(&onceStreamingInit, ^{
```

Comment:
`dispatch_once` shouldn't be needed here

Reply:
fixed